src/components/Calculator.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
+import Output from './Output';
+import Input from './Input';
+import calculate from '../logic/calculate';
+
+const Calculator = () => {
+  const [state, setState] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const handleClick = (e) => {
+    setState({ ...state, ...calculate(state, e.target.innerText) });
+  };
+
+  const { next, total } = state;
+
+  return (
+    <div className="grid container max-width-sm">
+      <Output value={next || total || '0'} />
+      <Input handler={handleClick} />
+    </div>
```

Comment:
- [ ] Your application looks nice already and works. However, when I clicked on an operator multiple times, I received this error: `Error: [big.js] Invalid number`. Please, let's implement an error handling exception to take care of this scenario.

Reply:
Why would you do that? Are you supposed to click on it multiple times? What's the whole point of you doing that?

Reply:
Lol. 😂 A user is free to use your application anyhow. It's your job as a developer to cover every edge cases.